Emby.XmlTv/Emby.XmlTv/Classes/XmlTvReader.cs

```diff
@@ -495,16 +495,32 @@ public void ProcessEpisodeNum(XmlReader reader, XmlTvProgram result)
                     ParseMovieDbSystem(reader, result);
                     break;
                 case "SxxExx":
-                    // TODO
-                    // <episode-num system="SxxExx">S03E12</episode-num>
-                    reader.Skip();
+                    ParseSxxExxSystem(reader, result);
                     break;
                 default: // Handles empty string and nulls
                     reader.Skip();
                     break;
             }
         }
+        public void ParseSxxExxSystem(XmlReader reader, XmlTvProgram result)
+        {
+            //// <episode-num system="SxxExx">S012E32</episode-num
+            string value = reader.ReadElementContentAsString();
+            Match res = Regex.Match(value, "s([0-9]+)e([0-9]+)", RegexOptions.IgnoreCase);
+            if (res.Success)
+            {
+                int parsedInt;
 
+                if (int.TryParse(res.Groups[1].Value, out parsedInt))
+                {
+                    result.Episode.Series = parsedInt;
+                }
+                if (int.TryParse(res.Groups[2].Value, out parsedInt))
```

Comment:
```suggestion

                if (int.TryParse(res.Groups[2].Value, out parsedInt))
```
Newline after closing brace.

Reply:
Is there a style guide published somewhere for this?

Reply:
Not yet, we do have some analyzers to enforce them.

Reply:
welll.. I believe the new function is in line with the rest of the file now.